be/src/runtime/load_channel.cpp

```diff
@@ -331,6 +331,16 @@ Status LoadChannel::_deserialize_chunk(const ChunkPB& pchunk, Chunk& chunk, fast
     return Status::OK();
 }
 
+std::vector<int64_t> LoadChannel::_get_index_ids() {
+    std::vector<int64_t> index_ids;
+    std::lock_guard l(_lock);
```

Comment:
How often is this fiction called? Will this impact the performance of concurrent ingestions